src/main/java/com/redhat/darcy/salesforce/RelatedList.java

```diff
@@ -107,8 +115,22 @@ public boolean isEmpty() {
         return noRows.isDisplayed();
     }
 
+    public boolean canShowMore() {
+        return showMore.isDisplayed();
+    }
+
+    public void clickShowMoreLink() {
+         after(showMore::click)
+            .expectCallTo(this::getRowCount, Matchers.greaterThan(getRowCount()))
+            .waitUpTo(2, ChronoUnit.MINUTES);
+    }
+    
+    public Locator byButtonTitle(String buttonTitle) {
```

Comment:
I think this should be protected, otherwise this is good.

I'm assuming usage is in subclasses in fields like, `private Button edit = button(byButtonTitle("Edit"));` ?